logs/plugin_bot.go

```diff
@@ -124,19 +124,14 @@ var cmdWhois = &commands.YAGCommand{
 		}
 
 		var memberStatus string
+		state := [6]string{"Playing", "Streaming", "Listening", "Watching", "Custom", "Competing"}
 		if !member.PresenceSet || member.PresenceGame == nil {
-			memberStatus = fmt.Sprintf("Has no active status, is invisible/offline or is not in the bot's cache.")
+			memberStatus = "Has no active status, is invisible/offline or is not in the bot's cache."
 		} else {
 			if member.PresenceGame.Type == 4 {
 				memberStatus = fmt.Sprintf("%s: %s", member.PresenceGame.Name, member.PresenceGame.State)
 			} else {
-				state := [4]string{"Playing", "Streaming", "Listening", "Watching"}
-				presenceName := "Unknown"
-				if member.PresenceGame.Type > 0 && len(state) > int(member.PresenceGame.Type) {
-					presenceName = state[member.PresenceGame.Type]
-				}
-
-				memberStatus = fmt.Sprintf("%s: %s", presenceName, member.PresenceGame.Name)
+				memberStatus = fmt.Sprintf("%s: %s", state[member.PresenceGame.Type], member.PresenceGame.Name)
```

Comment:
A combination of the two changes seem more future-proof to me.
```suggestion
				presenceName := "Unknown"
				if member.PresenceGame.Type > 0 && len(state) > int(member.PresenceGame.Type) {
					presenceName = state[member.PresenceGame.Type]
				}

				memberStatus = fmt.Sprintf("%s: %s", presenceName, member.PresenceGame.Name)
```

Reply:
Sounds good, gives us a grace period when Discord decides to ship a new presence type